test/cljc/spec_tools/swagger/core_test.cljc

```diff
@@ -324,6 +405,26 @@
                :path (st/create-spec {:spec (s/keys :req [::id])})
                :body (st/create-spec {:spec ::address})}}))))
 
+  (testing "::parameters with refs"
+    (is (=
+          {:parameters [{:in "body",
+                         :name "spec-tools.swagger.core-test/ref-spec",
+                         :description "",
+                         :required true,
+                         :schema {:$ref "#/definitions/RefSpec"}}],
+           :definitions {"RefSpec" {:type "object",
```

Comment:
Note: this `:definitions` is then lifted to the very top level of the swagger doc by

https://github.com/metosin/reitit/blob/15e0c95cb6e8ab1f653204f910dfb1851c0b0663/modules/reitit-swagger/src/reitit/swagger.cljc#L136

We could add a test that documents this feature end-to-end on the reitit side.